bap-vibes/tests/unit/test_parse_c.ml

```diff
@@ -77,7 +77,7 @@ let test_ite _ =
   assert_parse_eq
     "int cond_expr; if(cond_expr){goto l1;}else{goto l2;};"
     "{
-       if (cond_expr) {
+       if (cond_expr <> 0) {
```

Comment:
Do we no longer support the more natural looking condition?

Reply:
What you're seeing is the generated BIL translation of Core Theory. This reflects the stricter semantics of Core Theory as opposed to the rather relaxed "truth values" in the C semantics. So testing for `cond_expr` really means testing that it does not equal zero. I felt that it was important to make this explicit to respect the typing discipline of Core Theory.

Reply:
Understood - I should have bothered to understand the test case better :)  Thanks!